### DISCUSSION TITLE
Problems with G-Helper after installing a new battery

### DISCUSSION THREAD

**Discussion by bbbeton** [seerge/g-helper#1978] [General]
Hi! 

I wanted to star by saying I love G-Helper and it is with a heavy heart that i have to go back to using Armoury Crate - here's why. I installed a new battery on my laptop and I noticed that whenever I unplugged the AC or restarted the laptop, it would work really slowly for a couple of minutes and then suddenly start working just fine. I have a 2021 G14 and I've been using G-Helper for a while now and loving it, so I was very surprised to find after a few hours of searching for the issue, that it was G-Helper that slowed my computer down. Whenever i turned off the autostart of G-Helper, the computer would boot up and react to the unplugging just fine, but when i did start it, the laptop slowed down for a couple of minutes. Has anyone else had this issue? I would really like to go back to using G-Helper but this is too much of an inconvenience for me. 
Cheers

**Comment by seerge** [comment]
@bbbeton G-Helper is not slowing your computer down anyhow, cause it simply can't. Most likely you have hardware problems due to a different battery. Since you have ignored all rules and didn't include any logs  I can't help further.

https://github.com/seerge/g-helper/issues/new/choose

**Reply by seerge** [reply]
P.S. The differences you observe are highly likely due to different settings. If you don't start any app (AC or G-Helper) your computer by default runs in "Balanced" mode with all other default settings, and when you do start app - it would set a mode you have used last.

**Reply by bbbeton** [reply]
By AC I meant power supply. Sorry for ignoring rules, I couldn't find any, I just looked on how other posts looked. What logs do you have in mind? G-helper runs silent on battery - shouldn't it draw less power than the windows default balanced mode?

**Reply by seerge** [reply]
It depends on what settings you have inside Silent. 

Also on GPU mode (if you have Optimized then it would try to turn GPU off), and it could be that due to damaged hardware (capacitor) / battery replacement BIOS is not able to do that.

**Reply by bbbeton** [reply]
I see you've seen my post on reddit (: Like I said there, I got the damaged capacitator replaced. In Silent i have settings for best power efficiency and CPU boost disabled. The thing that puzzles me is why does the laptop work fine without any power management apps, but doesn't when i turn them on?

**Reply by seerge** [reply]
Then check what else is not default, check if GPU mode affects it, etc.

**Reply by bbbeton** [reply]
OK, thanks for all your help. Again, sorry for ignoring any rules. I love GHelper, keep up the good work (:
